pyhs3/zwave.py

```diff
@@ -126,6 +128,48 @@ async def dim(self, percent: int):
         await self._request('get', params=params)
 
 
+class ZWaveTemperature(HomeSeerDevice):
```

Comment:
Is it the case that the value from the HomeSeer API is always a string? If so then I think there is a more elegant way that I would want to handle this in the base HomeSeerDevice class that would make it easier for subclasses to specify whether the value is string or numeric and handle conversion to float/int in the base class automatically.

Reply:
Yes, the value is always a string since it is coming from ASCIIListener.  This could be handled in the HomeSeerDevice base class. The only reason I made the change as I did was to avoid any change to the base class.

Reply:
Removed 'value' override from zwave device classes and modified the value property of the base class as follows: 

    @property
    def value(self):
        if str(self._value).find(".") > -1:
            return float(self._value)
        else: 
            return int(self._value)

Reply:
Commit that to your branch and I'll make a slight tweak to it.

Reply:
The API always sends the DeviceValue (numeric) for the device. The text strings are the DeviceString for the device. They are not part of the ASCII API. 

Changes committed.